internal/namespaces/instance/v1/custom_server_rdp_test.go

```diff
@@ -0,0 +1,143 @@
+package instance_test
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	iamCLI "github.com/scaleway/scaleway-cli/v2/internal/namespaces/iam/v1alpha1"
+	"github.com/scaleway/scaleway-cli/v2/internal/namespaces/instance/v1"
+	iam "github.com/scaleway/scaleway-sdk-go/api/iam/v1alpha1"
+	"golang.org/x/crypto/ssh"
+)
+
+// tryLoadKey will try to load an RSA SSH Key from given path.
+// If not found, it will generate the key and create the file.
+func loadRSASSHKey(path string) (*rsa.PrivateKey, error) {
+	pemContent, err := os.ReadFile(path)
+	if err != nil && !os.IsNotExist(err) {
+		return nil, fmt.Errorf("failed to read test key: %w", err)
+	}
+
+	if *core.UpdateCassettes {
+		privateKey, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Is there a strong constraint on the size of the key? If so we should mention it in the documentation and have it as a constant to change it easily later on in case we need to increase the size of the key

Reply:
There is no constraint. This is a random value and my goal was to reduce file size.